zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraSpanStore.scala

```diff
@@ -374,10 +294,14 @@ class CassieSpanStore(
 
     // if we have a span name, look up in the service + span name index
     // if not, look up by service name only
-    val idx: ColumnFamily[String, Long, Long] =
-      spanName.map(_ => ServiceSpanNameIndex).getOrElse(ServiceNameIndex)
-    // TODO: endTs seems wrong here
-    idx.getRowSlice(key, Some(endTs), None, limit, Order.Reversed) map colToIndexedTraceId
+    pool {
+      (spanName match {
```

Comment:
i've left this. i can't think of an intuitive variable name here, hence the comment.

the closest variable name i can think of is timestampByTraceId, but i find that not intuitive enough for two lines of code.